tests/_others/sd_and_radio/sd_and_radio.ino

```diff
@@ -0,0 +1,52 @@
+#include <CanSatKit.h>
+#include <SPI.h>
+#include <SD.h>
+
+using namespace CanSatKit;
+
+int counter = 1;
+bool led_state = false;
+long last_blink;
+File file;
+
+Radio radio(10,
```

Comment:
use defined pins in library instead of magic numbers
```
Pins::Radio::ChipSelect,
Pins::Radio::DIO0,
```

Reply:
done